src/main/java/com/persado/oss/quality/stevia/selenium/core/controllers/factories/WebDriverWebControllerFactoryImpl.java

```diff
@@ -102,23 +111,34 @@ public String getBeanName() {
         return "webDriverController";
     }
 
-    private WebDriver getRemoteWebDriver(String rcUrl, Capabilities desiredCapabilities) {
-        WebDriver driver;
-        Augmenter augmenter = new Augmenter(); // adds screenshot capability to a default webdriver.
+    private WebDriver getRemoteWebDriver(String rcUrl, Capabilities desiredCapabilities) throws MalformedURLException, NoSuchFieldException, IllegalAccessException {
+        WebDriver driver = null;
+        //ReadTimeout when creating the RemoteWebDriver objects
+        //needs to take into consideration the time needed for
```

Comment:
Are these comments needed. I think you're explaining the same thing in the block below

Reply:
Removed